source/views/components/searchbar/index.android.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react'
+import {View} from 'react-native'
+import MaterialSearchBar from 'react-native-material-design-searchbar'
+
+type PropsType = {
+  placeholder?: string,
+  onChangeText: string => any,
+}
+
+export const SearchBar = () => <View />
```

Comment:
Temporarily displaying a blank view (suggested by @hawkrives) to stall off Android search. We want to nativify this as much as we can... which will involve rendering the searchbar inside the navigation component.